probe/core/src/main/java/org/jboss/weld/probe/JsonObjects.java

```diff
@@ -373,7 +373,7 @@ public int compare(ConfigurationKey o1, ConfigurationKey o2) {
                     }
                     desc = description.value();
                 }
-            } catch (Exception e) {
+            } catch (NoSuchFieldException e) {
```

Comment:
I know :-). My point is that we should not probably change the original functionality...